lib/DB/Schema/Result/UserSet.pm

```diff
@@ -155,7 +155,7 @@ __PACKAGE__->typecast_map(
 	}
 );
 
-my $set_type = {
+our $set_type = {
```

Comment:
Why is this changed to `our`?  Generally, in modern perl you don't want `our` at all.  Is this variable even used?  It doesn't look like it is.

Reply:
I'll look at this again.  There was some issue that it wasn't visible in a subclass I think.

Reply:
I get errors when it accesses line 166 of `Result::UserSet.pm`, for reverse mapping the number of the subclass with the type. 

If you have some other ways of doing this, I welcome them.